stock_lot_production_date/__manifest__.py

```diff
@@ -3,8 +3,9 @@
 {
     "name": "Stock Lot Production Date",
     "Summary": "Set a production date on lot.",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "author": "Camptocamp, Odoo Community Association (OCA)",
+    "maintainers": ["SodexisTeam"],
```

Comment:
@SodexisTeam I'd prefer an individual account

Reply:
I often forget to put myself in this field, I'm OK to be listed there too :+1:

Reply:
I could do that on all versions later myself, it's not a blocker for this migration.

Reply:
@sebalix We have a few days waiting for this change to be approved from Roussel

Could you change it?

Notice that the maintainers field enable the github user to merge

So, it could be better than only the person have this extra access

Reply:
@moylop260 Do you have contact with @SodexisTeam members ?

We could merge this quickly if the change is done.

Reply:
@rousseldenis @sebalix @moylop260 We have changed the maintainer to individual account as per your suggestion.  Thanks for your patience.